tests/integration/spring5/src/main/java/org/glassfish/jersey/server/spring/test/resource/AccountSpringResource.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) 2013, 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package org.glassfish.jersey.server.spring.test.resource;
+
+import org.glassfish.jersey.server.spring.test.AccountService;
+import org.glassfish.jersey.server.spring.test.HK2ServicePerLookup;
+import org.glassfish.jersey.server.spring.test.HK2ServiceRequestScoped;
+import org.glassfish.jersey.server.spring.test.HK2ServiceSingleton;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.core.MediaType;
+
+import java.math.BigDecimal;
+
+/**
+ * Spring managed JAX-RS resource for testing jersey-spring.
+ *
+ * @author Marko Asplund (marko.asplund at yahoo.com)
+ */
+@Path("/spring/account")
+@Singleton
+//@Component
+public class AccountSpringResource {
```

Comment:
@jansupol This is a problem I didn't find a way around yet, seems like with WebApplicationInitializer is Spring context started before HK2 bridge has anything to offer.